src/ut/cache_test.cpp

```diff
@@ -232,19 +287,41 @@ TEST_F(CacheInitializationTest, TransportException)
 TEST_F(CacheInitializationTest, NotFoundException)
 {
   cass::NotFoundException nfe;
-  EXPECT_CALL(_cache, get_client()).Times(1).WillOnce(Throw(nfe));
-  EXPECT_CALL(_cache, release_client()).Times(0);
+  EXPECT_CALL(_client, set_keyspace(_)).Times(1).WillOnce(Throw(nfe));
+  EXPECT_CALL(_resolver, success(_targets[0])).Times(1);
```

Comment:
Worth explaining in a comment why we call success even though `connection_test` returns `NOT_FOUND`.